ui/components/projects/ProjectTwoPictures.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react"
+import { Image } from "react-datocms"
+
+import { ICmsTwoPicturesBlockFragment } from "codegen-gql-types"
+import styles from "./projects.module.css"
+
+interface IProps {
+  data: ICmsTwoPicturesBlockFragment
+}
+
+export const ProjectTwoPictures: React.FC<IProps> = ({ data }) => {
+  const { leftPicture, rightPicture } = data
+
+  return (
+    <div className={styles.twoPictures}>
+      <Image
+        className={styles.twoPicturesItem}
+        data={leftPicture!.responsiveImage!}
+        intersectionMargin="500px"
+      />
+      <Image
+        className={styles.twoPicturesItem}
+        data={rightPicture!.responsiveImage!}
+        intersectionMargin="500px"
+      />
```

Comment:
От повторения можно избавиться, типа:

```suggestion
     [leftPicture!, rightPicture!].map({ responsiveImage } => 
       <Image className={style.twoPicturesItem} data={responsiveImage!} instersectionMargin="500px" />
```

Reply:
Добавил обертку ResponsiveImage